AtEventDisplay/AtViewerManager.h

```diff
@@ -69,13 +76,16 @@ class AtViewerManager final : DataHandling::AtObserver {
    DataHandling::AtTreeEntry &GetCurrentEntry() { return fEntry; }
    DataHandling::AtPadNum &GetPadNum() { return fPadNum; }
 
+   void SetCheckGood() { fCheckGood = true; }
+   void SetCheckBranch(TString name) { fCheckBranchName = name; }
+
```

Comment:
Using info from above this would be
```cpp
void SetCheckBranch(AtBranch &branch) { fCheckGood = true; fCheckEvt = new FairRootEventInfo<AtRawEvent>(branch);}
```